cmd/conformance-tester/pkg/runner/runner.go

```diff
@@ -356,7 +356,7 @@ func (r *TestRunner) executeTests(
 	healthCheck := func() error {
 		log.Info("Waiting for cluster to be successfully reconciled...")
 
-		return wait.PollLog(ctx, log, 5*time.Second, 5*time.Minute, func(ctx context.Context) (transient error, terminal error) {
+		return wait.PollLog(ctx, log, 5*time.Second, 10*time.Minute, func() (transient error, terminal error) {
```

Comment:
This doesn't compile.

Reply:
I had to add it at some point to fix a bug. I will remove and try without it.